libminifi/include/FlowController.h

```diff
@@ -68,21 +69,22 @@ class FlowController : public core::controller::ControllerServiceProvider, publi
    * Flow controller constructor
    */
   explicit FlowController(std::shared_ptr<core::Repository> provenance_repo, std::shared_ptr<core::Repository> flow_file_repo, std::shared_ptr<Configure> configure,
-                          std::unique_ptr<core::FlowConfiguration> flow_configuration, std::shared_ptr<core::ContentRepository> content_repo, const std::string name, bool headless_mode);
+                          std::unique_ptr<core::FlowConfiguration> flow_configuration, std::shared_ptr<core::ContentRepository> content_repo, std::string name, bool headless_mode);
```

Comment:
this should be updated in the cpp file, too

Reply:
No, this was intentional. Top-level `const` in the parameter list is ignored in the declaration, so I prefer no-`const` in declaration and `const` in the definition (when possible).